geode-core/src/distributedTest/java/org/apache/geode/cache30/CacheXMLPartitionResolver.java

```diff
@@ -43,7 +44,12 @@ public Serializable getRoutingObject(EntryOperation opDetails) {
   @Override
   public void close() {}
 
-  // @Override
+  @Override
+  public int hashCode() {
+    return Objects.hash(this.resolveProps);
```

Comment:
why are you using Objects.hash() instead of Objects.hashCode()?  The latter does less work.

Reply:
@bschuchardt I just redid what you did as part of GEODE-5986 in classes like Filter70 etc. I can change it to Objects.hashcode()